<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProteinModelFrame</class>
 <widget class="QFrame" name="ProteinModelFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QGroupBox" name="mol_rendermode_groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>150</width>
     <height>130</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="full_mol_rendermode_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Molecule</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="bb_mol_rendermode_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Backbone Only</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pocatoms_rendermode_radioButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pocket Atoms</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="none_rendermode_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="rendermodel_groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>180</width>
     <height>190</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="sf_model_radiobutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Space Fill</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="bs_model_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ball Stick</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ssf_model_radiobutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>130</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Small Space Fill</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="add_radius_sf_model_doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="add_radius_sf_model_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Additional Atom Radius</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="alpha_value_sf_model_doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="alpha_value_sf_model_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Value</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="showhide_surface_groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>150</width>
     <height>130</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="show_surface_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Surface</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="hide_surface_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide Surface</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="show_surface_wireframe_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Wireframe</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QtColorPicker" name="surface_color_colorpicker">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Surface Color</string>
    </property>
    <property name="useStandardColors" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="pocket_groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>150</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>Pockets</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="show_all_pockets_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>135</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show All Pockets</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="pocket_alpha_num_spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>58</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pocket_alpha_num_label">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>36</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha No.</string>
    </property>
   </widget>
   <widget class="QLabel" name="pocket_num_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>92</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pocket No.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pocket_num_spinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>58</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="alpha_complex_groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>180</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Alpha Complex</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="show_alpha_complex_tets_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Tetras</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_alpha_complex_tris_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Tris</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_alpha_complex_edges_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Eges</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_to_file_pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/proteinvis.qrc">
     <normaloff>:/images/save_all.png</normaloff>:/images/save_all.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="protein_grouping_comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>atom group by options</string>
   </property>
  </widget>
  <widget class="QTableView" name="protein_grouping_tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>449</y>
     <width>330</width>
     <height>181</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="reload_pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/proteinvis.qrc">
     <normaloff>:/images/reload2.png</normaloff>:/images/reload2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="save_trans_pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save Transformation</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/proteinvis.qrc">
     <normaloff>:/images/db_update.png</normaloff>:/images/db_update.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="load_trans_pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Load Transformation</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/proteinvis.qrc">
     <normaloff>:/images/db_comit.png</normaloff>:/images/db_comit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QPushButton</extends>
   <header>qtcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/proteinvis.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>show_all_pockets_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>pocket_num_spinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_all_pockets_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>pocket_num_label</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
